README.md

```diff
@@ -18,4 +18,16 @@ Then you need to run mkdocs from the root of that repository:
 
     $ mkdocs build
 
+Sometimes this might fail on Fedora 24+, so perform below steps
```

Comment:
@prashanthpai , I did fresh clone again and followed the steps again. But still getting the same error.

[psony@dhcp193-172 glusterdocs]$ cat /etc/fedora-release 
Fedora release 25 (Twenty Five)
[psony@dhcp193-172 glusterdocs]$ mkdocs --version
mkdocs, version 0.16.3
[psony@dhcp193-172 glusterdocs]$ pip --version
pip 8.1.2 from /usr/lib/python2.7/site-packages (python 2.7)
[psony@dhcp193-172 glusterdocs]$ mkdocs build
ERROR   -  Config value: 'docs_dir'. Error: The 'docs_dir' should not be the parent directory of the config file. Use a child directory instead so that the config file is a sibling of the config file. 
ERROR   -  Config value: 'site_dir'. Error: The 'site_dir' should not be within the 'docs_dir' as this leads to the build directory being copied into itself and duplicate nested files in the 'site_dir'.(site_dir: '/home/psony/glusterdocs/site', docs_dir: '/home/psony/glusterdocs') 

Aborted with 2 Configuration Errors!